bin/kano-updater-quickcheck

```diff
@@ -5,12 +5,27 @@
 # Copyright (C) 2015 Kano Computing Ltd.
 # License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
 #
+# Runs the updater in boot-window mode, if upgrades are avaiable.
+# Pass "--dry-run" to simply get an errorcode indication, 1 means updates are available.
+#
 
 STATUS_FILE=/var/cache/kano-updater/status.json
+dry_run=$1
 
-# check if we need to run 
-DO_RUN=$(jq '.is_scheduled!=0 or .state!="no-updates"' $STATUS_FILE)
+function are_updates_available()
+{
+    # check if we need to run
+    DO_RUN=$(jq '.is_scheduled!=0 or .state!="no-updates"' $STATUS_FILE)
+    if [ "$DO_RUN" == "true" ]; then
+        return 1
```

Comment:
Hmm, I didn't think that you could return from a bash function. I thought that the best that you could do was just set a variable and check it elsewhere.